src/shared/colors.ts

```diff
@@ -16,6 +16,7 @@ enum Colors {
   BLACK = '#000000',
   WHITE = '#FFFFFF',
   DARK_BACKGROUND = '#181818',
+  VALIDATION_ERROR = '#F14517',
```

Comment:
Please name it according to Color naming convention, like RED or RED_BRIGHT

Reply:
done